doc/api/inspector.md

```diff
@@ -108,6 +108,49 @@ with an error. [`session.connect()`] will need to be called to be able to send
 messages again. Reconnected session will lose all inspector state, such as
 enabled agents or configured breakpoints.
 
+## inspector.ws
+
+The Inspector can be interacted with via a websocket-based protocol. These APIs
+allow programatic control of the websocket, equivalent to use `--inspect*` CLI
+options.
+
+### ws.listen()
+
+Start the websocket listener.
+
+### ws.unlisten()
+
+Stop the websocket listener.
+
+### ws.id()
+
+Return the UUID string of the listener, or undefined if the listener is not
+started.
+
+### ws.host([newHost])
```

Comment:
technically, there are 2 objects here. the configuration, which can have a getter and setter (or a unified function like I did here), and that actual running listener (if it is running). its not possible to change the port on the fly, the current listener would have only a getter. So, there could be 3 functions... or one, like here, and a warning that the new value won't take effect until a new listener is started. Thoughts?